Justfile

```diff
@@ -0,0 +1,30 @@
+
+CPU_IMAGEBASENAME := "tvm"
+GPU_IMAGEBASENAME := "tvm-gpu"
+REGISTRYREPO := "octoml"
+REGISTRY := "docker.io"
```

Comment:
Up until now, i had just been referencing this image as `octoml/tvm` ... does this change make it so we need to FQ it with `docker.io/octoml/tvm`?

Reply:
My impression was the image was being implicitly resolved, see Docker Hub.